docker-compose-swarm.yml

```diff
@@ -5,7 +5,7 @@ services:
     ports:
       - "2181:2181"
   kafka:
-    image: ${REPO}kafka-docker:latest
+    image: kafka:latest
```

Comment:
This should be `wurstmeister/kafka:latest` (so it references the docker hub build)